src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -3376,6 +3376,67 @@ def test_aks_nodepool_snapshot(self, resource_group, resource_group_location):
         delete_snapshot_cmd = "aks nodepool snapshot delete --resource-group {resource_group} --name {snapshot_name} --yes --no-wait"
         self.cmd(delete_snapshot_cmd, checks=[self.is_empty()])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(
+        random_name_length=17, name_prefix="clitest", location="westus2"
+    )
+    def test_aks_nodepool_delete_machines(self, resource_group, resource_group_location):
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=86906639&view=results) to validate the change. If test passed, find the recording file from pipeline artifacts and commit it to your branch.

Reply:
https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=86968601&view=results

Pipeline run and succeed.